phoenix-core/src/main/java/org/apache/hadoop/hbase/ipc/controller/MetadataRpcController.java

```diff
@@ -36,7 +37,16 @@
 			.add(PhoenixDatabaseMetaData.SYSTEM_CATALOG_NAME)
 			.add(PhoenixDatabaseMetaData.SYSTEM_STATS_NAME)
 			.add(PhoenixDatabaseMetaData.SYSTEM_SEQUENCE_NAME)
-			.add(PhoenixDatabaseMetaData.SYSTEM_FUNCTION_NAME).build();
+			.add(PhoenixDatabaseMetaData.SYSTEM_FUNCTION_NAME)
+            .add(SchemaUtil.getPhysicalTableName(PhoenixDatabaseMetaData.SYSTEM_CATALOG_NAME_BYTES, true)
```

Comment:
is the namespace mapping enabled always for system tables (which is what I am inferring from true being passed from isNamespaceMapped) ? Looking at QueryServicesOptions.java, looks like it isn't.